customer_service/index.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+header("Content-Type: application/json");
+header("Access-Control-Allow-Origin: *");
+header("access-control-allow-origin: *");
+header("Access-Control-Allow-Methods: GET,HEAD,OPTIONS,POST,PUT");
+header("access-control-allow-methods: GET,HEAD,OPTIONS,POST,PUT");
+header("Access-Control-Allow-Headers: Or_+_igin, X-Requested-With, Content-Type, Accept, Authorization,Access-Control-Allow-Origin,access-control-allow-headers");
+header("access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept, Authorization,Access-Control-Allow-Origin,access-control-allow-headers,access-control-allow-methods");
+
+error_reporting(E_ALL);
+ini_set('display_errors',1);
+
+require_once "vendor/autoload.php";
+
+use Lib\ErrorReporter\ErrorReporter;
+use Customer\Controllers\ApplicationController;
+use Customer\Controllers\CustomerController;
+use Customer\Controllers\CustomerSessionController;
+
+//instantiating and configuring ErrorReporter
+ErrorReporter::getInstance();
+ErrorReporter::setErrorTraceMode(1);
+ErrorReporter::setResponseMessageType(0);
+
+$applicationController = new ApplicationController();
+$decoded = [];
+
+//Process POST | PUT | DELETE request.
+if(in_array($_SERVER['REQUEST_METHOD'], array('POST', 'PUT', 'DELETE'))) {
+    
+    //Make sure that the content type of the request has been set to application/json
+    $contentType = isset($_SERVER["CONTENT_TYPE"]) ? trim($_SERVER["CONTENT_TYPE"]) : '';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [$_SERVER data not unslashed before sanitization. Use wp_unslash() or similar](https://app.codacy.com/gh/BuildForSDG/Team-064-Backend/pullRequest?prid=5616370)